src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Debugging/DefaultLSPBreakpointSpanProvider.cs

```diff
@@ -58,7 +58,7 @@ internal class DefaultLSPBreakpointSpanProvider : LSPBreakpointSpanProvider
             var languageResponse = response?.Response;
             if (languageResponse is null)
             {
-                _logHubLogger.LogInformation("The breakpoint position could not be mapped to a valid range.");
+                _logHubLogger?.LogInformation("The breakpoint position could not be mapped to a valid range.");
```

Comment:
Can we apply `https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.codeanalysis.membernotnullattribute?view=net-6.0` to `InitializeLogHubAsync` to avoid this?

Also, curious why this didn't give a warning before.

Reply:
I actually tried the membernotnull attribute on the initializeasync method; however, because initializeasync is async it complains about the inner await resulting in the exiting of the method. i.e.:
![image](https://user-images.githubusercontent.com/2008729/163240033-2df58c68-a284-43da-82b4-13ab2447b85d.png)